client/client-core/src/test/java/com/cognifide/aet/common/ProcessorFactoryTest.java

```diff
@@ -28,94 +27,63 @@
 import org.mockito.runners.MockitoJUnitRunner;
 
 import javax.jms.JMSException;
-import javax.jms.Message;
-import javax.jms.ObjectMessage;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.instanceOf;
 import static org.hamcrest.Matchers.nullValue;
 import static org.hamcrest.core.Is.is;
-import static org.mockito.Mockito.when;
 
 
 @RunWith(MockitoJUnitRunner.class)
 public class ProcessorFactoryTest {
 
   private static final String REPORT_URL = "http://cognifide.com";
 
-  private static final String BUILD_DIR = "/";
-
-  @Mock
-  private Message plainMessage;
-
-  @Mock
-  private ObjectMessage message;
-
   @Mock
   private RunnerTerminator runnerTerminator;
 
-  @Mock
-  private ProcessingErrorMessage processingErrorMessage;
-
-  @Mock
-  private FinishedSuiteProcessingMessage reportMessage;
-
-  @Mock
-  private ProgressMessage progressMessage;
-
   @Before
   public void setUp() {
 
   }
 
   @Test
-  public void processTest_messageIsNull() throws JMSException {
-    MessageProcessor processor = ProcessorFactory.produce(null, REPORT_URL, null, runnerTerminator);
-
-    assertThat(processor, is(nullValue()));
-  }
-
-  @Test
-  public void processTest_messageIdNotAnObject() throws JMSException {
-    MessageProcessor processor = ProcessorFactory.produce(plainMessage, REPORT_URL, null, runnerTerminator);
+  public void processTest_statusIsNull() throws JMSException {
+    StatusProcessor processor = ProcessorFactory.produce(null, REPORT_URL, null, runnerTerminator);
```

Comment:
It would be more readable if you named the null values and made it explicit if they matter in this scenario.